Controller/Checkout.php

```diff
@@ -348,6 +351,6 @@ protected function processGroupedItems($quote,array $data)
             $simpleProduct = $this->catalogProductFactory->create()->loadByAttribute('sku',$item['sku']);
             $this->addItemToQuote($quote,$simpleProduct,$simpleProduct,[],[],$item['quantity']);
         }
-        $this->checkoutSession->setBreadItemAddedToQuote(true);
+        $this->customerSession->setBreadItemAddedToQuote(true);
```

Comment:
what was the reason for this change from checkoutSession to customerSession everywhere? like what is it changing/why